src/legacy/core_plugins/vis_default_editor/public/components/agg_params_helper.ts

```diff
@@ -109,19 +125,27 @@ function getAggParamsToRender({ agg, editorConfig, metricAggs, state }: ParamIns
         metricAggs,
         state,
         value: agg.params[param.name],
+        schemas,
       });
     }
   });
 
   return params;
 }
 
+const filterByName = propFilter('name');
+
 function getAggTypeOptions(
   agg: IAggConfig,
   indexPattern: IndexPattern,
-  groupName: string
+  groupName: string,
+  allowedAggs: string[]
 ): ComboBoxGroupedOptions<IAggType> {
-  const aggTypeOptions = aggTypeFilters.filter((aggTypes as any)[groupName], indexPattern, agg);
+  const aggTypeOptions = aggTypeFilters
```

Comment:
Since the `aggTypeFilters` is used only in `rollup` for adding a filter,
makes sense to get rid of it at all.
Please, take a look at this comment https://github.com/elastic/kibana/pull/57695/files/555e19e6ecf08b05e0dc39ebdc92d9f2d5bf32ef#r380614658

Reply:
yes, lets do that in a followup